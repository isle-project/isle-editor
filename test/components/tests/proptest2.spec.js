// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import PropTest2 from '@isle-project/components/tests/proptest2';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<PropTest2 />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <PropTest2
			data={heartdisease}
			var1="Drugs"
			success={2}
			group="Complications"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
