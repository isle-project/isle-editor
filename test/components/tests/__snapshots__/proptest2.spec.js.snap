// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PropTest2 /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    proptest2-title-grouped
  </label>
  <br />
  <span>
    Let 
    <div />
     be the population probability of 
    <code>
      Drugs
    </code>
     being 
    <code />
     in the first group,
    <br />
    and
     
    <div />
     
    probability-second-group
  </span>
  <div />
  <span>
     
    vs
     
  </span>
  <div />
  <pre>
    Two-sample z-test

Alternative hypothesis: True difference in proportions is not equal to 0

    pValue: 0.2059
    statistic: -1.2651
    95% confidence interval: [-0.1971,0.0425]


    <br />
    sample-proportion-in-group
     "
    0
    ": 
    0.109
    <br />
    sample-proportion-in-group
     "
    1
    ": 
    0.186
  </pre>
</div>
`;
